### PR TITLE
docs(readme): add link to the contribution guidelines

### PR DESCRIPTION
**Pull request by maxime-clem**
## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [X] I've confirmed the [contribution guidelines].
- [X] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The Reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
-# autoware-documentation
+# Autoware documentation
 
 - This repository contains the [MkDocs](https://www.mkdocs.org/) source files for the [Autoware documentation hosted on GitHub Pages](https://autowarefoundation.github.io/autoware-documentation/main/).
 - Since Autoware is made up of [multiple repositories](https://github.com/autowarefoundation/), a central documentation repository is important to make information accessible from a single place.
 - For more information about Autoware and its related repositories, refer to the [Autoware Foundation's organization profile](https://github.com/autowarefoundation/.github/blob/main/profile/README.md).
+
+## Contributions
+
+To contribute to this repository, see the [corresponding guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/documentation-guidelines/).
```
Comment (kenji-miyake) [inline README.md:9]: I think the URL is not appropriate.

```suggestion
To contribute to this repository, see the [corresponding guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
```

Or this one?

```suggestion
To contribute to this repository, see the [contribution guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/).
```

Comment (kenji-miyake) [reply README.md:9]: Or if you intended how to write documents: 

```suggestion
To contribute to this repository, see the [documentation guidelines](https://autowarefoundation.github.io/autoware-documentation/main/contributing/documentation-guidelines/).
```

Comment (maxime-clem) [reply README.md]: Thank you. I meant only the documentation guidelines.

**Review by kenji-miyake — APPROVED** [review]
Thanks!

**Pull request merged**